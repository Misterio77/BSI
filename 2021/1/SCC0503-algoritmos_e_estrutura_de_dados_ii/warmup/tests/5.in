data/reg.dat
5
97
