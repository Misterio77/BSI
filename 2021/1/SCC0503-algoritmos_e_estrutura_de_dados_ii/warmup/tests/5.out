nUSP: 97
Nome: Chester C. Spencer
Curso: Ciencia da Computacao
Nota: 7.00
